### DISCUSSION TITLE
Remap Windows+Backtick to use terminal?

### DISCUSSION THREAD

**Discussion by RandyTiddMSFT2** [microsoft/terminal#14892] [Q&A] [answered]
Windows + ` (backtick) keyboard shortcut pulls down a powershell 5.1 prompt.  Is there a way to have this launch terminal instead?

**Comment by zadjii-msft** [comment, marked as answer]
Excuse me _what_? When you press the "Quake Mode" keybinding to open the quake window, the Terminal should just use whatever your default profile is. For most folks, that's just Windows PowerShell. But "Terminal" isn't a shell (like cmd / powershell), the Terminal is just a hosting window for all those shells.

What shell were you expecting to open?

**Reply by DHowett** [reply]
(I think Mike is surprised by his reading that _PowerShell_ is opening without being inside Terminal;) is that so?

**Reply by RandyTiddMSFT2** [reply]
Thanks for providing the name "Quake Mode", suddenly I can search for information about this feature. :)  I can now see it's running windows terminal.  I'm just setting up Windows Terminal to start using it more, and I mistakenly believed the windows+backtick feature was inbox and just launching the default powershell shell.  

Changing the default profile in Terminal solved part of my issue.  I wanted it to launch powershell 7 instead of 5, and now it does.  Thanks.

As for what I'd _like_ "Quake Mode" to do:

-Launch the tabbed interface so I can have multiple profiles in Quake Mode.

-It only launches if Windows Terminal was already running.  Is that expected?  It would be nice if this launched terminal on its own.

Actually, I think the same is covered in this idea as 1,2, and 4:

#14414

**Reply by zadjii-msft** [reply]
You also may be interested in the `globalSummon` keybinding. It's a bit more general-purpose than the "easter egg" quake mode one. 

> Launch the tabbed interface so I can have multiple profiles in Quake Mode

It already does support multiple tabs, the _quake window just opens in "focus mode" by default. 

Most of the rest of your requests are linked up to #8888, including
* #9996
* #9992 (for making _quake not open in focus mode by default)

**Reply by RandyTiddMSFT2** [reply]
Thanks, either globalSummon or the newly enabled command palette to toggle focus mode in the quake window should work for me.

I did encounter an issue in the FAQ though here:

[How do I set the size of the quakeMode window?](https://github.com/microsoft/terminal/wiki/Frequently-Asked-Questions-(FAQ)#how-do-i-set-the-size-of-the-quakemode-window)

The sample action has:

{ "keys": "win+`", "command": { "action": "globalSummon", "dropdownDuration": 200, "toggleVisibility": true, "monitor": "toCursor", "desktop": "toCurrent" } }

The monitor command is "toCursor", but apparently only "any", "toMouse", and "toCurrent" are valid.

Thanks again for the help.
